src/main/java/org/onebusaway/alexa/lib/ObaClientSharedCode.java

```diff
@@ -44,7 +46,7 @@
      * (http://developer.onebusaway.org/modules/onebusaway-application-modules/current/api/where/index.html),
      * which we will then use to get stop and arrival data for that region.
      *
-     * @param l geographic location used to search for nearby regions
+     * @param l                          geographic location used to search for nearby regions
```

Comment:
Extra whitespace, please remove

Reply:
This was added by IntelliJ to autoindent to match the indent for the parameter below.
Will remove extra whitespaces